Source/Core/Core/HotkeyManager.cpp

```diff
@@ -36,7 +36,7 @@ const std::string hotkey_labels[] = {
     _trans("Start Recording"),
     _trans("Play Recording"),
     _trans("Export Recording"),
-    _trans("Read-only mode"),
+    _trans("Read-Only mode"),
```

Comment:
This seems wrong to me, since "mode" is still lowercase. The hyphenated parts should only be capitalized if it's in title case.